fcrepo-http-api/src/test/java/org/fcrepo/integration/http/api/FedoraHtmlIT.java

```diff
@@ -104,7 +104,7 @@ public void testGetRootTemplate() throws IOException {
         method.addHeader(ACCEPT, "text/html");
         try (final CloseableHttpResponse response = execute(method)) {
             final String html = EntityUtils.toString(response.getEntity());
-            assertTrue(contains(html, "class=\"fcrepo_root\""));
+            assertTrue(html, contains(html, "class=\"fcrepo_root\""));
```

Comment:
not that i mind this, but was this intentionally left in?

Reply:
This was part of @awoods PR on this branch. Now I'm not sure so much.